calyx-opt/src/analysis/mod.rs

```diff
@@ -17,6 +17,7 @@ pub mod reaching_defns;
 mod read_write_set;
 mod schedule_conflicts;
 mod share_set;
+mod static_par_domination;
```

Comment:
Is there ever a case we'd want to run `static_par_domination` separately from the full domination pass?